core-java-modules/core-java-11/src/test/java/com/baeldung/java11/httpclient/test/HttpRequestUnitTest.java

```diff
@@ -48,7 +48,12 @@ public void shouldUseHttp2WhenWebsiteUsesHttp2() throws IOException, Interrupted
         assertThat(response.version(), equalTo(HttpClient.Version.HTTP_2));
     }
 
-    @Test
+    /*
+    * This test will fail as soon as the given URL returns a HTTP 2 response.
+    * Therefore, let's leave it commented out.
+    * */
+
+   /* @Test
```

Comment:
can we use the `@Disabled` annotation instead?

Reply:
ok